ui/src/keyboardMappings/KeyboardLayouts.ts

```diff
@@ -0,0 +1,218 @@
+import { keysUKApple, charsUKApple, modifiersUKApple } from './layouts/uk_apple';
```

Comment:
Keys and modifiers correspond to definitions in the [Linux USB HID gadget driver](https://www.kernel.org/doc/Documentation/usb/gadget_hid.txt) and are not language-dependent. These definitions could probably be factored out for easier maintenance.